plugins/inputs/modbus/README.md

```diff
@@ -0,0 +1,82 @@
+# Telegraf Input Plugin: Modbus
+
+The Modbus plugin collects Discrete Inputs, Coils, Input Registers and Holding Registers via Modbus TCP or Modbus RTU/ASCII
+
+### Configuration:
+
+```toml
+ ## Connection Configuration
+ ##
+ ## The module supports connections to PLCs via MODBUS/TCP or
+ ## via serial line communication in binary (RTU) or readable (ASCII) encoding
+ ##
+ 
+ ## Slave ID - addresses a MODBUS device on the bus
+ ## Range: 0 - 255 [0 = broadcast; 248 - 255 = reserved]
+ slave_id = 1
+ 
+ ## Timeout for each request
+ timeout = "1s"
+ 
+ # TCP - connect via Modbus/TCP
+ controller = "tcp://localhost:502"
+ 
+ # Serial (RS485; RS232)
+ #controller = "file:///dev/ttyUSB0"
+ #baud_rate = 9600
+ #data_bits = 8
+ #parity = "N"
+ #stop_bits = 1
+ #transmission_mode = "RTU"
+ 
+ 
+ ## Measurements
+ ##
+ 
+ ## Digital Variables, Discrete Inputs and Coils
+ ## name    - the variable name
+ ## address - variable address
+ 
+ discrete_inputs = [
+   { name = "Start",          address = [0]},   
+   { name = "Stop",           address = [1]},
+   { name = "Reset",          address = [2]},
+   { name = "EmergencyStop",  address = [3]},
```

Comment:
Can you update the sample configuration to use snake_case for the field/tag names?  This will better match other Telegraf plugins.

Reply:
done ✔️